frontend/templates/views/partials/example-sidebar.j2

```diff
@@ -20,7 +20,7 @@
                 <div class="nav-link">
                   <span>{{ category }}</span>
                 </div>
-                <input class="nav-trigger" type="checkbox" checked>
+                <input class="nav-trigger" type="checkbox" {% if doc.category == category %}checked{% endif %}>
```

Comment:
qq: why not use an accordion here? It'd have the advantage that it maintains expansion state across page loads.

Reply:
Oh that's nice, I wasn't aware. Since this would be a change not just the sample sidebar but the one from the Guides section would also benefit from I'd suggest to do update this in an upcoming PR. Does `amp-accordion` work seamlessly with `amp-sidebar` though?

Reply:
It should work with amp-sidebar. Another PR sgtm.